Libraries/Components/Picker/__tests__/Picker-test.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ * @emails oncall+react_native
+ * @flow
+ */
+
+'use strict';
+
+const React = require('React');
+const Picker = require('../Picker');
+
+const render = require('../../../../jest/renderer');
+
+describe('<Picker />', () => {
+  it('should render as <View> when mocked', () => {
+    const instance = render.create(
+      <Picker selectedValue="foo" onValueChange={jest.fn()}>
```

Comment:
Should we be abstracting this out into a constant at the top? It’s pretty important that they are consistent across all of the tests, right? Probably should do that for all of the tests.